homeassistant/components/satel_integra/__init__.py

```diff
@@ -45,20 +48,42 @@
 ZONE_SCHEMA = vol.Schema({
     vol.Required(CONF_ZONE_NAME): cv.string,
     vol.Optional(CONF_ZONE_TYPE, default=DEFAULT_ZONE_TYPE): cv.string})
+EDITABLE_OUTPUT_SCHEMA = vol.Schema({vol.Required(CONF_ZONE_NAME): cv.string})
+PARTITION_SCHEMA = vol.Schema(
+    {vol.Required(CONF_ZONE_NAME): cv.string,
+     vol.Optional(CONF_ARM_HOME_MODE, default=DEFAULT_CONF_ARM_HOME_MODE):
+     vol.In([1, 2, 3]),
+     }
+    )
+
+
+def is_alarm_code_necessary(value):
+    """Check if alarm code must be configured."""
+    if not isinstance(value, dict):
```

Comment:
This shouldn't be necessary as the validator before this one in the schema already checks this.

Reply:
True, removed.